docs/pages/demo/datepicker/ServerRequest.example.jsx

```diff
@@ -26,7 +26,7 @@ function ServerRequest() {
       <DatePicker
         label="With server data"
         value={selectedDate}
-        onChange={handleDateChange}
+        onChange={date => handleDateChange(date)}
```

Comment:
I'm wondering if we shouldn't make it clear that we don't expect a returned value.

```suggestion
        onChange={date => {
          handleDateChange(date)
        }}
```

Reply:
I dont think it will help somebody. I think 99% of js developers will be confused by such examples